sirepo/template/srw.py

```diff
@@ -1898,17 +1898,39 @@ def _remap_3d(info, allrange, z_label, z_units, width_pixels, scale='linear'):
         except Exception:
             pkdlog('Cannot resize the image - scipy.ndimage.zoom() cannot be imported.')
             pass
-    z_label = z_label
-    if z_units:
-        z_label += ' [' + z_units + ']'
+    # rotate 3D image    
+    if rotate_angle:
+        rotate_reshape = (rotate_reshape == "1")
+        try:
+            from scipy import ndimage
+            pkdlog('Size before: {}  Dimensions: {}', ar2d.size, ar2d.shape)
```

Comment:
Rather than using pkdlog() for debugging information like this, you can use pkdc() which only shows up if you turn on tracing for the file.

For example, if you change it to pkdc() and run the server like this:

```shell
$ export PYKERN_PKDEBUG_CONTROL=srw.py
$ sirepo service http
```

The debug statements will now appear, but they won't appear unless tracing is enabled. That way, on production, the items won't get printed in the logs.

Reply:
Thanks for the "debugging" updates!